Doc/library/sqlite3.rst

```diff
@@ -1111,15 +1136,13 @@ Row Objects
 
 .. class:: Row
 
-   A :class:`Row` instance serves as a highly optimized
+   A :class:`Row` instance that serves as a highly optimized
```

Comment:
Maybe the original was better.

Reply:
Yeah, the revised wording doesn't really make much sense—it implies a ``:class:`Row`` _class_ (the title of the section, and thus the implied subject) is a ``:class:`Row`` _instance_, which is not what you intend.

```suggestion
   A :class:`Row` instance serves as a highly optimized
```